keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -638,24 +640,168 @@ always dispatched immediately.  Following is how the other requests
 are dispatched at a given apiserver.
 
 The concurrency limit of an apiserver is divided among the non-exempt
-priority levels in proportion to their assured concurrency shares.
-This produces the assured concurrency value (ACV) for each non-exempt
-priority level:
+priority levels, and higher ones can do a limited amount of borrowing
+from lower ones.
```

Comment:
We need to adjust this - it's all-to-all, right?

My concern with this approach is that I'm conceptually not a fan of allowing random low-level PLs to borrow from e.g. leader-election PL. Let' me read the other two proposals now.